openquake/hazardlib/imt.py

```diff
@@ -268,6 +268,18 @@ def LiqProb():
     """
     return IMT('LiqProb')
 
+def LiqOccur():
+    """
+    Liquefaction occurrence class
+    """
+    return IMT('LiqOccur')
+
+def LSE():
+    """
+    Liquefaction spatial extent
```

Comment:
will be good to mention the units (is it % of the grid cell or area represented by each site?)